src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  flex-shrink: 0;
+}
+
+.nav_link {
+  display: block;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #000;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: #00ACDC;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.logo {
+  font-size: 0;
+}
```

Comment:
Redundant
```suggestion
```

Reply:
if i deleted this class with this style, I have a problem with npm test...